src/main/typescript/applications/item/weaponSheet.ts

```diff
@@ -24,6 +24,7 @@ export default class WeaponSheet extends WvItemSheet {
     defaultOptions.height = 500;
     defaultOptions.width = 800;
     return foundry.utils.mergeObject(defaultOptions, {
+      // TODO? Should this be changed too? Having a tough time understanding what this code does
```

Comment:
This determines which HTML elements can be dragged on the sheet. This was used in the past to allow dragging weapon attack buttons into the hotbar. Though we would need to refactor this anyway, because for players at least we would need to add a check that the weapon is also readied when they execute the attack.

Reply:
But for now I should still change the html class, correct?

Reply:
You still need to change the CSS class correct. But this should not be affected by this change. The drag selectors are from the point of view of the application element root, not from the overall page document root.

This means that the selector here is already inside of the element that is affected by your CSS changes.